addons/sale/models/sale_order.py

```diff
@@ -37,7 +37,7 @@
 
 class SaleOrder(models.Model):
     _name = 'sale.order'
-    _inherit = ['portal.mixin', 'product.catalog.mixin', 'mail.thread', 'mail.activity.mixin', 'utm.mixin']
+    _inherit = ['portal.mixin', 'product.catalog.mixin', 'mail.thread', 'mail.activity.mixin', 'utm.mixin', 'account.invoice.import.mixin']
```

Comment:
Could have generic name as it is not only for `account.invoice` model :eyes:

Also we could move `display_inactive_currency_warning` in mixin as it will be useful for sale order also and in purchase order in future (We didn't added in sale order at that time because we don't want to duplicate too much code from accounting :sweat_smile: ). WDYT?
```suggestion
    _inherit = ['portal.mixin', 'product.catalog.mixin', 'mail.thread', 'mail.activity.mixin', 'utm.mixin', 'account.import.mixin']
```

Reply:
The name isn't perfect but `account.import.mixin` could be confused with importing other accounting data (e.g. accounts).
I wanted to find a name that reflected the fact that this mixin is for importing business documents. But `account.business.document.import.mixin` would have been too long and a bit confusing given Odoo's terminology. 

Thus, even though it's not just for invoices, I think the name is okay, but I'm open to more suggestions!

For the `display_inactive_currency_warning`, its compute method needs to access the move's `state` field - so it already looks like something which might be better handled separately in sale orders, even if that involves duplication :sweat_smile:

Maybe at some point it could be nice to have some more commonality between the different business documents, but I'm not sure we can do it in this PR :joy: